<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grade_sel</class>
 <widget class="QWidget" name="grade_sel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩统计</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>学生个人成绩统计</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>63</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入您的学号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_sel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>128</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>统计</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>课程成绩统计</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入课程号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="coure_sel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>128</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>统计</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
